src/main/java/org/myrobotlab/service/SwingGui.java

```diff
@@ -477,7 +477,11 @@ public void setStatus(Status inStatus) {
       status.setOpaque(false);
     }
 
-    status.setText(inStatus.detail);
+    if (inStatus.detail != null && inStatus.detail.length() > 128) {
+      status.setText(inStatus.detail.substring(128));

```

Comment:
it seems like we should have a different approach to displaying these warnings so they don't resize the swing gui when long errors show up.  Also. the arbitrary length of 128 will still resize on low resolution monitors  (like the raspi touch screen.)

Reply:
ya .. but that means someone needs to come up with an alternative. Having a stack trace go through will certainly blow it out at any resolution, so this was a stop gap until someone come up with a better plan.

Reply:
maybe changing the status bar to be something like a text area with scroll bars and word wrap ?  I'm fine with the truncation now, but this still probably needs to be revisited.